src/MentionsTextInput.js

```diff
@@ -248,7 +248,7 @@ export default class MentionsTextInput extends Component {
 
     const start = this.triggerMatrix[index][0];
     const end = this.triggerMatrix[index][1];
-    const pattern = new RegExp(`${this.props.trigger}[a-z0-9_-]*`, `gi`);
+    const pattern = new RegExp(`${this.props.trigger}[a-zA-Z0-9_-]*`, `gi`);
```

Comment:
this.props.trigger doesnt get escaped so if it was `.` then it would be constantly triggering the mentions. `(` would probably cause a crash. It doesnt look like theres an included regex.escape function though so it might not be worth it to fix